sympy/solvers/solveset.py

```diff
@@ -969,6 +969,10 @@ def _solveset(f, symbol, domain, _check=False):
     given symbol."""
     # _check controls whether the answer is checked or not
     from sympy.simplify.simplify import signsimp
+    from sympy.logic.boolalg import BooleanTrue
+
+    if isinstance(f, BooleanTrue):
+        return domain
```

Comment:
This line isn't covered by the tests according to [this report](https://codecov.io/gh/sympy/sympy/pull/19986/diff). Can you please add some tests?

Reply:
Do you mean something like this?
```
    f = BooleanTrue()
    assert solveset(f, x, domain=Interval(-3, 10)) == Interval(-3, 10)
```

Reply:
Yes. This should work.